### DISCUSSION TITLE
Does consult-fd support directory abbreviation?

### DISCUSSION THREAD

**Discussion by fountainer** [minad/consult#934] [Q&A] [answered]
I have some directory abbreviations in `directory-abbrev-alist `. `consult-recent-file` can use the list to abbreviate directories. But `consult-fd` cannot do this. Does consult-fd support directory abbreviation? or I'm missing something?

**Comment by minad** [comment]
No directory-abbrev-list is not supported. But consult-fd shows the paths relative to the current directory or default directory.

**Reply by fountainer** [reply]
I have some files in my iCloud driver.  The paths are usually very long. I made some soft links for those directories with long names.  `consult-fd` shows the full path on my computer, not relative to current directory or default directory. The front part of the path is not useful and the later part may be not fully displayed.

**Reply by minad** [reply, marked as answer]
Can you please try to reproduce this problem with a minimal configuration? See https://github.com/minad/consult?tab=readme-ov-file#bug-reports. It looks as if consult-fd-args is modified in your configuration to generate absolute paths.

**Reply by fountainer** [reply]
You are right. `consult-fd-args` is modified by doom-emacs. Thanks.
